content/docs/handling-events.md

```diff
@@ -91,35 +91,35 @@ ReactDOM.render(
 );
 ```
 
-[**Try it on CodePen**](http://codepen.io/gaearon/pen/xEmzGg?editors=0010)
+[**Experimente no CodePen**](http://codepen.io/gaearon/pen/xEmzGg?editors=0010)
 
-You have to be careful about the meaning of `this` in JSX callbacks. In JavaScript, class methods are not [bound](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_objects/Function/bind) by default. If you forget to bind `this.handleClick` and pass it to `onClick`, `this` will be `undefined` when the function is actually called.
+Você precisa ter cuidado com o significado do `this` nos callbacks do JSX. Em JavaScript, os métodos de classe não são [vinculados](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_objects/Function/bind) por padrão. Se você esquecer de fazer o bind de `this.handleClick` e passá-lo para um `onClick`, o `this` será `undefined` quando a função for realmente chamada.
 
-This is not React-specific behavior; it is a part of [how functions work in JavaScript](https://www.smashingmagazine.com/2014/01/understanding-javascript-function-prototype-bind/). Generally, if you refer to a method without `()` after it, such as `onClick={this.handleClick}`, you should bind that method.
+Este não é um comportamento específico do React; é uma parte de [como funcionam as funções em JavaScript](https://www.smashingmagazine.com/2014/01/understanding-javascript-function-prototype-bind/). Geralmente, se você referir a um método sem `()` depois dele, como `onClick={this.handleClick}`, você deve fazer o bind manual deste método.
 
-If calling `bind` annoys you, there are two ways you can get around this. If you are using the experimental [public class fields syntax](https://babeljs.io/docs/plugins/transform-class-properties/), you can use class fields to correctly bind callbacks:
+Se ficar chamando "bind" incomoda você, há duas maneiras de contornar isso. Se você estiver usando a [sintaxe experimental de campos de classe pública](https://babeljs.io/docs/plugins/transform-class-properties/), você pode usar campos de classe para vincular callbaks corretamente:
 
 ```js{2-6}
 class LoggingButton extends React.Component {
-  // This syntax ensures `this` is bound within handleClick.
-  // Warning: this is *experimental* syntax.
+  // Essa sintaxe garante que o `this` seja vinculado ao handleClick.
```

Comment:
I'd rather don't translating some technical words, but that's not the case. Sometimes during this page, we use `bind` in English and other times we translate the word binding. What about or translation every `bind` term? Maybe we'll need to introduce that `bind` means `vincular`. I'm thinking about people with no English knowledge. This can confuse their minds.

Reply:
https://github.com/reactjs/pt-BR.reactjs.org/issues/4#issuecomment-461766907

Reply:
@eduardomoroni vou responder em português: quando a gente ver que o 'bind' no contexto da frase se refere ao método, não iremos traduzir o bind. Se não, traduziremos. Será uma questão de entender o contexto da frase. Ex.: [MDN Docs - Bind](https://developer.mozilla.org/pt-BR/docs/Web/JavaScript/Reference/Global_objects/Function/bind)

Reply:
Então eu deixo do jeito que está ou eu corrijo?

Reply:
> Esse bind é necessário para fazer o `this` funcionar dentro do callback

> Aqui utilizamos o `bind` para que o `this` funcione dentro da nossa callback

Acredito que não precise seguir a tradução ao pé da letra, escrever dessa forma mantem o uso da nomeclatura da função e deixa claro o motivo dela

Reply:
I'll set this review as resolved as I think it is. please rollback if I'm mistaken